miniscope_io/models/stream.py

```diff
@@ -20,7 +19,7 @@ class StreamBufferHeaderFormat(BufferHeaderFormat):
     :class:`~miniscope_io.stream_daq.StreamDaq`
     """
 
-    pixel_count: Range
+    pixel_count: int
```

Comment:
does the parent model no longer need to be `Union[int, Range]` as well?

Reply:
Yes, regarding the streamDaq part, those can all be `int`. I didn't touch the parent model because I wasn't sure how the SD card module uses it.

Reply:
fixed in https://github.com/Aharoni-Lab/miniscope-io/pull/29/commits/994ca2c4377423862f6c9d504ea82ee74ef70e13